config.go

```diff
@@ -112,3 +117,24 @@ func (c *Config) InitDefault(log *zap.Logger) {
 		c.Resource.ServiceNamespaceKey = fmt.Sprintf("RoadRunner-%s", uuid.NewString())
 	}
 }
+
+func setClientFromEnv(client *Client, log *zap.Logger) {
+	// https://opentelemetry.io/docs/specs/otel/protocol/exporter/#specify-protocol
+	exporterEnv := "OTEL_EXPORTER_OTLP_TRACES_PROTOCOL"
+	exporterVal := os.Getenv(exporterEnv)
+	if exporterVal == "" {
+		exporterEnv = "OTEL_EXPORTER_OTLP_PROTOCOL"
+		exporterVal = os.Getenv(exporterEnv)
+	}
+	switch exporterVal {
+	case "":
```

Comment:
The same here: fallthrough ?

Reply:
As with the other switch, if the switch value is empty, we don't want to set a value for the client, so the empty case is intentional. I'll add a `// do nothing` comment here too.